src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -262,139 +314,159 @@
                 <div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row vads-u-margin-top--2">
                   <p class="vads-u-font-weight--bold vads-u-margin--0 vads-u-margin-right--2">Where</p>
                   <a href="" rel="noreferrer noopener" target="_blank">
-                    Not sure what to put here
+                    TODO
```

Comment:
Assuming this is something for a later task?

Reply:
Yeah, I think there might be a few pieces of logic going into that about rendering a physical vs virtual location and I think this PR is busy enough